capstone/src/main/webapp/script.js

```diff
@@ -162,6 +163,7 @@ function precinctControl(controlDiv, map) {
       document.getElementById('chart-title').textContent =
         'Sentiment Percentages in ' + precinct;
       loadCharts();
+      loadWordcloud();
```

Comment:
Are you sure you want to load the word cloud everytime the precinct button is selected? This could  lead to increased latency.